flexget/plugins/input/discover.py

```diff
@@ -181,7 +193,16 @@ def estimated(self, entries, estimation_mode):
         result = []
         for entry in entries:
             est_date = estimator.estimate(entry)
-            if est_date is None:
+            if est_date is False:
```

Comment:
Should not use `is` with boolean

Reply:
eh, he actually should do that @paranoidi 🙃

Reply:
I needed to do that because I'm returning None or False, and I have to check what is the one! None is the default return, False is to inform that it has no information to the series! I know what you are going to say.... wait for it... not a very elegant way to test this... :)